.github/workflows/test.yml

```
@@ -113,7 +113,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Install nightly Rust
-        run: rustup default nightly
+        run: rustup default nightly-2024-05-10
```

I think it would be good to remind ourselves to remove this. Otherwise others will likely think that this is there for a specific reason.

Let's hope it's fixed in a month :smile: 

```suggestion
        # TODO: Unpin this after 2024-06-10
        run: rustup default nightly-2024-05-10
```

---

I just merged an update to `time` dep in https://github.com/google/mdbook-i18n-helpers/pull/198 and the workaround is no longer necessary.

---

I switched back to "nightly" only